### PR TITLE
Upvotes/downvotes

### PR DESCRIPTION
**Pull request by Dusk-Argentum**
Added support for upvotes/downvotes, and all changes necessary to support them. Call with `~upvote`/`~downvote` or `~up`/`~down`

### CODE REVIEW

File: lib/reports.py [hunk 1]
```diff
@@ -13,7 +13,9 @@
 VERI_EMOJI = {
     1: "\u2705",  # WHITE HEAVY CHECK MARK
     -1: "\u274c",  # CROSS MARK
-    0: "\u2139"  # INFORMATION SOURCE
+    0: "\u2139",  # INFORMATION SOURCE
+    2: "\u2b06", # UPVOTE
+    3: "\u2b07" # DOWNVOTE
```
Comment (mommothazaz123) [inline lib/reports.py]: maybe change this to -2 so positive veris are positive, and negative are negative

File: lib/reports.py [hunk 2]
```diff
@@ -89,6 +91,17 @@ def canrepro(self, author, msg):
         }
         self.verification += 1
         self.attachments.append(attachment)
+        
+    def upvote(self, author, msg):
+        if[a for a in self.attachments if a['author'] == author and a['veri']]:
```
Comment (mommothazaz123) [inline lib/reports.py]: will raise this exception if a user has added cr/cnr already

**Comment by Dusk-Argentum** [comment]
Changes made as requested

File: lib/reports.py [hunk 3]
```diff
@@ -1,172 +1,172 @@
...
+            raise Exception("You have already upvoted this report.")
+        attachment = {
+            'author': author,
+            'msg': msg,
+            'veri': 2
+        }
+        self.verification += 2
```
Comment (mommothazaz123) [inline lib/reports.py]: verification should only go up by 1 on an up/downvote

File: lib/reports.py [hunk 4]
```diff
@@ -1,172 +1,172 @@
...
+
+PRIORITY = {
+    -2: "Patch Pending", -1: "Resolved",
+    0: "P0: Critical", 1: "P1: Very High", 2: "P2: High", 3: "P3: Medium", 4: "P4: Low", 5: "P5: Trivial",
+    6: "Pending/Other"
+}
+VERI_EMOJI = {
+    -2: "\u2b07" # DOWNVOTE
+    -1: "\u274c",  # CROSS MARK
+	0: "\u2139",  # INFORMATION SOURCE
```
Comment (mommothazaz123) [inline lib/reports.py]: inconsistent use of spaces/tabs

**Pull request merged**